addons/account/static/src/js/section_and_note_fields_backend.js

```diff
@@ -1,106 +1,67 @@
+/** @odoo-module **/
 
-odoo.define('account.section_and_note_backend', function (require) {
-// The goal of this file is to contain JS hacks related to allowing
-// section and note on sale order and invoice.
+import { registry } from "@web/core/registry";
+import { ListRenderer } from "@web/views/list/list_renderer";
+import { X2ManyField } from "@web/views/fields/x2many/x2many_field";
+import { TextField } from "@web/views/fields/text/text_field";
+import { CharField } from "@web/views/fields/char/char_field";
 
-// [UPDATED] now also allows configuring products on sale order.
+const { Component } = owl;
 
-"use strict";
-var FieldChar = require('web.basic_fields').FieldChar;
-var FieldOne2Many = require('web.relational_fields').FieldOne2Many;
-var fieldRegistry = require('web.field_registry');
-var ListFieldText = require('web.basic_fields').ListFieldText;
-var ListRenderer = require('web.ListRenderer');
-
-var SectionAndNoteListRenderer = ListRenderer.extend({
+export class SectionAndNoteListRenderer extends ListRenderer {
     /**
-     * We want section and note to take the whole line (except handle and trash)
-     * to look better and to hide the unnecessary fields.
+     * The purpose of this extension is to allow sections and notes in the one2many list
+     * primarily used on Sales Orders and Invoices
      *
      * @override
      */
-    _renderBodyCell: function (record, node, index, options) {
-        var $cell = this._super.apply(this, arguments);
-
-        var isSection = record.data.display_type === 'line_section';
-        var isNote = record.data.display_type === 'line_note';
+    setup() {
+        super.setup();
+        this.allowSectionsAndNotes = true;
+    }
 
-        if (isSection || isNote) {
-            if (node.attrs.widget === "handle") {
-                return $cell;
-            } else if (node.attrs.name === "name") {
-                var nbrColumns = this._getNumberOfCols();
-                if (this.handleField) {
-                    nbrColumns--;
-                }
-                if (this.addTrashIcon) {
-                    nbrColumns--;
-                }
-                $cell.attr('colspan', nbrColumns);
-            } else {
-                $cell.removeClass('o_invisible_modifier');
-                return $cell.addClass('o_hidden');
-            }
-        }
+    get isSectionOrNote() {
```

Comment:
Do we support the `get` JS feature?
I think remember Sam told me we couldn't use that because of browser compatibility

Reply:
https://github.com/odoo/odoo/blob/master/addons/web/static/src/views/list/list_renderer.js#L191
No list renderer, no section and note 🤷